src/Gofer-Tests/GoferReferenceTest.class.st

```diff
@@ -162,46 +180,53 @@ GoferReferenceTest >> testVersionShouldFindWorkingCopy [
 	| versionReference workingCopy |
 	versionReference := GoferVersionReference name: 'Gofer-Core-lr.18'.
 	workingCopy := versionReference workingCopy.
-	self assert: workingCopy packageName = 'Gofer-Core'
+	self assert: workingCopy packageName equals: 'Gofer-Core'
 ]
 
 { #category : #tests }
 GoferReferenceTest >> testVersionShouldParseComplexName [
 	| queryReference |
+	
 	queryReference := GoferVersionReference name: 'Seaside2.8b5'.
-	self assert: queryReference packageName = 'Seaside2'.
-	self assert: queryReference author isEmpty.
-	self assert: queryReference branch = '8b5'.
-	self assert: queryReference versionNumber = 0.
+	self 
+		assert: queryReference packageName equals: 'Seaside2';
```

Comment:
I (with Denis) think that we should not use cascade on assertions. It makes debugging harder on failing assertions.

Reply:
Why does it make debugging harder? And if so - shouldnt we then change Debugging?